src/utils/runtime/src/logger.rs

```diff
@@ -451,12 +457,24 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
                         // https://github.com/jaegertracing/jaeger-ui/issues/336
                         format!("{}-{}", settings.name, id),
                     ),
-                    KeyValue::new(resource::SERVICE_INSTANCE_ID, id),
+                    KeyValue::new(resource::SERVICE_INSTANCE_ID, id.clone()),
                     KeyValue::new(resource::SERVICE_VERSION, env!("CARGO_PKG_VERSION")),
                     KeyValue::new(resource::PROCESS_PID, std::process::id().to_string()),
                 ])))
                 .install_batch(sdk::runtime::Tokio)
-                .unwrap()
+                .unwrap();
+
+            let exporter = opentelemetry_otlp::new_exporter()
```

Comment:
What's the difference between this exporter with the one in L449? Can we reuse them?